systemtests/bin/download_sym_files.py

```diff
@@ -77,13 +73,15 @@ def download_sym_files(base_url, csv_file):
 
             # Download the file
             headers = {"User-Agent": "tecken-systemtests"}
-            resp = requests.get(url, headers=headers, timeout=60)
+            resp = http_get_with_retry(url, headers=headers, timeout=60)
 
             for item in resp.history:
                 if item.status_code in (301, 302):
                     click.echo(f">>> redirect: {item.url}")
             click.echo(f">>> final: {resp.url}")
             click.echo(f">>> status code: {resp.status_code}")
+            if resp.status_code == 200:
+                click.echo(f">>> file size: {len(resp.content):,} bytes")
```

Comment:
This gives some helpful. context to why it took so long to download the file.

Reply:
f-strings can do a lot of amazing things, but the docs are terrible. Then I found this which is really helpful for me:

https://fstring.help/cheat/